src/firebase.ts

```diff
@@ -59,14 +69,14 @@ const getLocationDict = async (collectionName: string) => {
     // `reduce` is a method that takes an array and reduces it to a single value
     const locationDict = docs.reduce((locationDict: FirebaseDict, doc: FirestoreDoc) => {
         const name = doc[FIRESTORE_FIELDS.NAME];
-        const originalLocation = doc[FIRESTORE_FIELDS.ORIGINAL_LOCATION];
+        const originalLocation = doc[FIRESTORE_FIELDS.ORIGINAL_LOCATION] ?? doc[FIRESTORE_FIELDS.RECIPE_PATH] ?? "";
```

Comment:
Ah I believe the config name isn't displaying is because I updated the database to use the new key name `source_path` instead of `original_location` while working on another ticket. I'll submit a small renaming PR to merge into this one. Apologies!

Reply:
honestly, we probably don't need to keep track of original location at all anymore. I'll double check, but I think I can just remove this